### PR TITLE
Added php.executablePath to package.json contributes.configuration

### PR DESCRIPTION
**Pull request by lokken**
Addresses https://github.com/felixfbecker/vscode-php-intellisense/issues/103

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -57,12 +57,22 @@
...
         "php.memoryLimit": {
           "type": "string",
           "default": "-1",
           "description": "The memory limit of the php language server. [Number][K|M|G]. Use '-1' to allow unlimited use of the RAM(default).",
           "pattern": "^\\d+[KMG]?$"
+        },
+        "php.executablePath": {
+          "type": [
+            "string",
+            "null"
```
Comment (felixfbecker) [inline package.json]: why null?

Comment (jens1o) [inline package.json]: Otherwise it won't be able to change something when the extension is installed  but not used.

Comment (lokken) [inline package.json]: Allows it to be set to null if PHP 7 is already available in the path? I mimicked the php.validate.executablePath configuration in the vscode codebase.

Comment (lokken) [inline package.json]: Without it, when you copy the setting into user or workspace settings, a green line is displayed.
screen shot 2017-04-14 at 12 38 46 pm

Comment (felixfbecker) [inline package.json]: but you can just omit the setting in that case. Having another special value kind besides `undefined` kind of implies a special meaning for it.

Comment (lokken) [inline package.json]: Understood, I was simply following the lead of the vscode php extension: https://github.com/Microsoft/vscode/blob/0c8a6d5ad4f1950407edfecfdd815d6f2cb74b01/extensions/php/package.json

Comment (lokken) [inline package.json]: I can remove it if you like.

Comment (felixfbecker) [inline package.json]: I see, you might want to override it in workspace settings. We can keep it

**Comment by Glutnix** [comment]
Any idea when this will make it into a release?

**Comment by felixfbecker** [comment]
Just did

**Pull request merged**